src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
Of course I did. What I mean is that

    if (UNIX)

is sufficient.

What we had before essentially was

    if (LINUX)

which I agree is now wrong.